File: src/my_project/easy_problems/from251to300/remove_duplicates_sorted_array.py
from typing import List 

class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        
        len_nums = len(nums)
        
        if len_nums == 0 or len_nums == 1:
            return len_nums
        
        j = 0
        for i in range(len_nums-1):
            if nums[i] != nums[i+1]:
                nums[j] = nums[i]
                j+=1
        nums[j] = nums[len_nums-1]
            
        return j + 1
                

File: my_project_typescript/easy_problems_typescript/from151to200/removeDuplicatesSortedArray.ts
function removeDuplicates(nums: number[]): number {

    let lenNums: number = nums.length

    if (lenNums === 0 || lenNums === 1){
        return lenNums
    }

    let k: number = 0
    for (let i = 0; i < lenNums-1 ; i++){
        if (nums[i] !== nums[i+1]){
            nums[k] = nums[i]
            k += 1
        }
    }
    nums[k] = nums[lenNums-1]
    return k+1

};